trend_micro_vision_one_xdr/README.md

```diff
@@ -1,43 +1,65 @@
-# Agent Check: trend-micro-vision-one-xdr
-
 ## Overview
 
-This check monitors [trend-micro-vision-one-xdr][1].
+[Trend Micro Vision One XDR][1] collects and automatically correlates data across multiple security layers: email, endpoint, server, cloud workload, and network. This allows for faster detection of threats and improved investigation and response times through security analysis.
 
-## Setup
+This integration ingests the following logs:
 
-### Installation
+- Workbench Alerts
+- Observed Attack Techniques
 
-The trend-micro-vision-one-xdr check is included in the [Datadog Agent][2] package.
-No additional installation is needed on your server.
+This integration collects all the above listed logs and sends them to Datadog for analysis. Datadog uses the built-in logs pipeline to parse and enrich these logs, enabling effortless search and analysis. The integration provides insight into workbench alerts and observed attack techniques through the out-of-the-box dashboards.
```

Comment:
Please add that we also offer out of the box detection rules for Ccloud SIEM customers.

Reply:
Done